source/lmp/pppm_dplr.cpp

```diff
@@ -109,8 +109,13 @@ void PPPMDPLR::compute(int eflag, int vflag)
   //   to fully sum contribution in their 3d bricks
   // remap from 3d decomposition to FFT decomposition
 
+#if LAMMPS_VERSION_NUMBER>=20210831
+  gc->forward_comm(GridComm::KSPACE,this,1,sizeof(FFT_SCALAR),REVERSE_RHO,
```

Comment:
Should be `reverse_comm`
```suggestion
  gc->reverse_comm(GridComm::KSPACE,this,1,sizeof(FFT_SCALAR),REVERSE_RHO,
```